src/Umbraco.Web.UI.Client/src/views/templates/edit.html

```diff
@@ -0,0 +1,175 @@
+<div ng-controller="Umbraco.Editors.Templates.EditController as vm">
+
+    <umb-load-indicator ng-if="vm.page.loading"></umb-load-indicator>
+
+   <form name="contentForm"
+      ng-submit="vm.save()"
+      novalidate
+      val-form-manager>
+
+
+        <umb-editor-view ng-if="!vm.page.loading">
+
+            <umb-editor-header
+                name="vm.template.name"
+                alias="vm.template.alias"
+                hideDescription="true"
+                description="vm.template.virtualPath"
+                description-locked="true"
+                menu="vm.page.menu"
+                hide-icon="false">
+            </umb-editor-header>
+
+            <umb-editor-container>
+
+                <div class="flex" style="margin-bottom: 30px;">
+
+                    <div class="flex">
+
+                        <div ng-class="{'btn-group umb-era-button-group': vm.template.masterTemplateAlias}" style="margin-right: 10px;">
+                            
+                            <button
+                                type="button"
+                                class="umb-era-button umb-button--s"
+                                ng-click="vm.openMasterTemplateOverlay()">
+                                <i class="icon icon-layout"></i>
+                                <span class="bold">Master template:</span> <span style="margin-left: 5px;">{{ vm.getMasterTemplateName(vm.template.masterTemplateAlias, vm.templates) }}</span>
+                            </button>
+
+                            <a ng-if="vm.template.masterTemplateAlias" ng-click="vm.removeMasterTemplate()" class="umb-era-button umb-button--s dropdown-toggle umb-button-group__toggle">
+                                <span class="icon icon-wrong"></span>
+                            </a>
+
+                        </div>
+                        
+                    </div>
+
+                    <div class="flex" style="margin-left: auto;">
+                        <div class="btn-group umb-era-button-group dropdown" style="margin-right: 10px;">
+                            
+                            <button
```

Comment:
Any plans to support other keyboard shortcuts to quickly insert stuff @madsrasmussen as we only have CTRL+S for saving the template.